modules/terraform/cmd.go

```diff
@@ -78,9 +87,9 @@ func GetExitCodeForTerraformCommand(t *testing.T, additionalOptions *Options, ar
 func GetExitCodeForTerraformCommandE(t *testing.T, additionalOptions *Options, additionalArgs ...string) (int, error) {
 	options, args := GetCommonOptions(additionalOptions, additionalArgs...)
 
-	logger.Log(t, "Running terraform %v", args)
+	logger.Log(t, "Running terragrunt with args", args)
```

Comment:
NIT: Oh I missed this in the first pass. This should probably reference `options.TerraformBinary`. Sorry about that!